landsat/anaconda-project.yml

```diff
@@ -10,38 +11,30 @@ examples_config:
 
 user_fields: [examples_config]
 
-channels: [pyviz, defaults]
+channels: [conda-forge]
 
 packages: &pkgs
-- python=3.7
-- notebook
-- bokeh
-- cartopy
-- datashader
-- geoviews
-- holoviews
-- numpy
-- pandas=1.2.5
-- rasterio
-- xarray
-- numba
-- param
-- nbsmoke=0.2.8
-- pytest=4.4.1
+- python >=3.10.11
+- notebook >=6.5.4
+- bokeh >=3.1.1
+- cartopy >=0.20.2
+- datashader >=0.14.4
+- geoviews >=1.10.0
+- holoviews >=1.16.0
+- numpy >=1.24.3
+- pandas >=2.0.2
+- rasterio >=1.2.10
+- xarray >=2023.5.0
+- numba >=0.57.0
+- param >=1.13.0
+- rioxarray >=0.14.1
+- jupyter_client =7.4.9
     
```

Comment:
These were generated by finding the minimum version from the lock file. I don't know if that's the best process but the lock file should still work for creating the environments.

Reply:
If it worked on defaults before, can you record here why conda-forge is needed now?  Presumably everything involved must be available on defaults, just not in compatible versions?

Reply:
And can you remind us why it needs these pins? I think it was something to do with cross-platform compatibility? Usually we'd put pins like that just to indicate the minimum version needed when the notebook was written, but this is a notebook from 2016 and so presumably none of those versions are truly minimal.

Reply:
> And can you remind us why it needs these pins? 

Isn't it just because we asked for it in a recent discussion here? As even if that's not required, that makes maintaining the project easier on the long term.

> the minimum version needed when the notebook was written

That might just be lost? I'm happy with updated pins.

Reply:
> Isn't it just because we asked for it in a recent discussion here? As even if that's not required, that makes maintaining the project easier on the long term.

Yes, that was my understanding. But in light of comments here: https://github.com/holoviz-topics/examples/issues/275 - I'm going to remove all the pins except the one for `jupyter_client`